File: find-numbers-with-even-number-of-digits/find-numbers-with-even-number-of-digits.go
func findNumbers(nums []int) int {
    cnt := 0
    for _, n := range nums {
        fmt.Println(len(string(n)))
        if (len(strconv.Itoa(n)) % 2) == 0 {
            cnt++
        }
    }
    
    return cnt
}

File: find-numbers-with-even-number-of-digits/find-numbers-with-even-number-of-digits.java
class Solution {
    public int findNumbers(int[] nums) {
        int cnt = 0;
        for (int num : nums) {
            String numeric = Integer.toString(num);
            if ((numeric.length() % 2) == 0) {
                cnt++;
            }
        }

        return cnt;
    }
}